LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ADR/LoRAADRManagerFactory.cs

```diff
@@ -42,13 +42,15 @@ private static LoRaADRInMemoryStore CurrentInMemoryStore
 
                 lock (InMemoryStoreLock)
                 {
+#pragma warning disable IDE0074 // Use compound assignment
 #pragma warning disable CA1508 // Avoid dead conditional code
                     // False positive.
                     if (inMemoryStore == null)
```

Comment:
Why not
``` csharp
inMemoryStore ??=  new new LoRaADRInMemoryStore();
```

Reply:
It generates this error : 

![image](https://user-images.githubusercontent.com/9513635/182321320-768c0d7c-15c6-4826-8efc-ee28010f9a2a.png)

We can also dismiss this error in CodeQL...